src/hooks/useTokenIdentityRegistryAgents.ts

```diff
@@ -34,6 +37,33 @@ export function useTokenIdentityRegistryAgents(registry?: EvmAddress) {
     }
   }, [registry]);
 
+  const filteredNotAgentsYetQueryResult = useQueries({
+    queries: (identityItems ?? []).map((agent) => ({
+      queryKey: [QueryKeys.ReadAgentInRegistry, agent],
+      enabled: !!identityItems?.length && !!registry,
```

Comment:
`identityItems` are not used in the query itself, I don't think we need to add them to the enabled flag. If we have stuff - we load it, otherwise `queries` would be empty anyway.